README.md

```diff
@@ -0,0 +1,85 @@
+# Twitter Driver
+
+A [Membrane.io](https://membrane.io/) driver for Twitter.
+
+## Actions
+
+Get [Client key and secret](https://developer.twitter.com/en/portal) from twitter dev portal and setup your [Callback URI](https://developer.twitter.com/en/docs/apps/callback-urls) with the program Endpoint URL.
+
+$~~~~$`mctl action "twitter:configure(CLIENT_ID:'<string>',CLIENT_SECRET:'<string>')"`
+
+Create a tweet (it also support polls!)
+
+$~~~~$`mctl action "twitter:tweet(text:'Hi!')"`
```

Comment:
It would be nice to separate this into two examples `Create a tweet` and `Create a poll tweet`